### DISCUSSION TITLE
2022-09-12 PyAerocom Meeting

### DISCUSSION THREAD

**Discussion by dulte** [metno/pyaerocom#729] [Development meetings]
## Last meeting

?

## PRs and Issues

- #691 
  - deals with #604
  - 90% done, no change
- #664 
  - deals with #663
  - status?
- #721 
- #727 
  - New colocation class. 3D, point cloud

## Other issues

- #699 
  - volunteers?
- #652
- #625 
  - waiting for feedback
- #728

**Comment by lewisblake** [comment]
Can we also discuss the fake-PR #727?

**Reply by dulte** [reply]
Sure :)
